packages/devtools_app/lib/src/common_widgets.dart

```diff
@@ -79,6 +79,64 @@ TextStyle primaryColorLight(TextStyle style, BuildContext context) {
   );
 }
 
+/// Button to pause.
+///
+/// * `includeTextWidth`: The minimum width the button can be before the text is
+///    omitted.
+/// * `onPressed`: The callback to be called upon pressing the button.
+class PauseButton extends StatelessWidget {
+  const PauseButton({
+    Key key,
+    this.includeTextWidth,
+    @required this.onPressed,
+  }) : super(key: key);
+
+  final double includeTextWidth;
```

Comment:
Nit: a bit of duplicate code here.
Create an IconLabelButton base class and then these classes can both just be one liners specifying the 
icon and text.